src/_guides/libraries/writing-package-pages.md

```diff
@@ -0,0 +1,294 @@
+in the following screenshot:
+
+{% asset libraries/package-page-sections.png alt="package page contains sections like package layout, flutter favorite, package scoring, verified publishers, pubspec file" class="screenshot" %}
+
+For details about other parts of the package page,
+follow these links:
+
+1. [Package layout ](https://dart.dev/tools/pub/package-layout)
+2. [Flutter Favorite](https://flutter.dev/docs/development/packages-and-plugins/favorites)
+3. [Package scoring](https://pub.dev/help/scoring)
+4. [Verified publishers](https://dart.dev/tools/pub/verified-publishers)
+5. [Pubspec file](https://dart.dev/tools/pub/pubspec)
+
+
+## Writing a good README is important
+
+Most package users who landed on your package on pub.dev will
+quickly scan your README to decide whether to try your package or not.
+A nice README will catch users’ attention and
+show that your package is worth trying.
+
+There are seven tips that you can follow to write a good README. 
+
+1. Write a short description at the top of the README
+2. Include visual content to explain things
+3. Utilize lists to present key information
+4. Include at least one usage example
+5. Use Dart code formatting
+6. Mention terms that people might use to search for
+7. Tell users where they can go next
+
+Throughout this document, we’ll use the README of the package
+[`in_app_purchase`][] as a “full” example.
+Depending on the complexity of your package,
+you can choose to go as simple as the README of the [`yaml`][] package. 
+
+{{ site.alert.note }}
+  Your README will be used in multiple places and contexts,
+  such as API references, in addition to the package page on pub.dev. 
+{{ site.alert.end }}
+
+## Seven tips for good README
+
+### 1. Write a short description at the top of the README
+
+According to our user research,
+package users spend only a few seconds to
+read the package description and make a decision whether to
+read the rest of the README or not.
+Thus, you should concisely describe what the package does or achieves,
+at a glance.
+Spend some time to craft a short and sweet description and
+help the user to make decisions.
+There’s no need to write the package name again on the top. 
+
+See examples of good descriptions below.
+
+*   A Flutter plugin for showing rainbows.
+*   This plugin allows Flutter apps to categorize bird sounds using machine learning.
+
+If there is important information users should know about,
+such as project status or constraints, note that at the top too. 
+
+*   Does not work on iOS versions below 10.3.
+
+If you have any badges, put them under the description or
+anywhere high on the page as well. 
+
+The package [`in_app_purchase`][]
+starts its README with a brief explanation of the package and a caution.
+
+{% asset libraries/package-page-description.png alt="description of the package in_app_purchase" class="screenshot" %}
+
+
+### 2. Include visual content to explain things
+
+UI is difficult to explain in text.
```

Comment:
This makes it seem like visual content is only useful for UI-related plugins. I'll think about how to fix it...